### PR TITLE
junonum: community keymap improvement

### PR DESCRIPTION
**Pull request by JunoNgx**
## Description
* Add fall back for `AG_NORM`
* Add handling of RGB
* Reindent codes
* Fix handling of Muse and audio

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by waffle87 — APPROVED** [review]

**Comment by JunoNgx** [comment]
This is a fix necessary for another one of my PRs:
https://github.com/qmk/qmk_firmware/pull/19735

File: layouts/community/ortho_4x12/junonum/keymap.c [hunk 1]
```diff
@@ -120,39 +123,38 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
+        QD_NOTE(_AS5), Q__NOTE(_C6),
+        Q__NOTE(_AS5), Q__NOTE(_C6)
+    };
 #endif
 
 #ifndef BACKLIGHT_ENABLE
-layer_state_t layer_state_set_user(layer_state_t state) {
-    return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
-}
+    layer_state_t layer_state_set_user(layer_state_t state) {
```
Comment (drashna) [inline layouts/community/ortho_4x12/junonum/keymap.c:149]: Might make more sense to use: 

```c
layer_state_t layer_state_set_user(layer_state_t state) {
    state = update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
#ifdef BACKLIGHT_ENABLE
    switch (get_highest_layer(state)) {
        case _QWERTY:
            backlight_set(0);
            break;
        case _FN1:
            backlight_set(3);
            break;
        case _DPAD:
            backlight_set(3);
            break;
    }
#endif
    return state;
}
```

File: layouts/community/ortho_4x12/junonum/keymap.c [hunk 2]
```diff
@@ -120,39 +123,38 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
-};
-
-float dpad_song_off[][2] = {
-	ED_NOTE(_C6), ED_NOTE(_C6),
-	ED_NOTE(_C6), ED_NOTE(_C6),
-	W__NOTE(_REST), QD_NOTE(_GS5),
-	QD_NOTE(_AS5), Q__NOTE(_C6),
-	Q__NOTE(_AS5), Q__NOTE(_C6)
-};
+    float plover_song[][2]    = SONG(PLOVER_SOUND);
```
Comment (drashna) [inline layouts/community/ortho_4x12/junonum/keymap.c:126]: you may want to run `qmk format-c` on this file, as it should format everything for you

Comment (JunoNgx) [reply layouts/community/ortho_4x12/junonum/keymap.c:126]: It flattens the layers and de-indent all the preprocessor directives, which is the opposite of I'm trying to fix. Thanks for the suggestion regardless.

Comment (JunoNgx) [reply layouts/community/ortho_4x12/junonum/keymap.c:126]: For this particular line, I'm trying align the assigned value with the next line.

File: layouts/community/ortho_4x12/junonum/keymap.c (hunk 1 above)
Comment (JunoNgx) [reply layouts/community/ortho_4x12/junonum/keymap.c]: Thanks, great suggestion. Amended.

**Review by drashna — APPROVED** [review]

**Pull request merged**